chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -293,6 +293,9 @@ pane.item.related.count.singular = %S related:
 pane.item.related.count.plural  = %S related:
 pane.item.parentItem			= Parent Item:
 
+drag.invalid.title 				= Invalid item
```

Comment:
Maybe move "Invalid Item" to something like `general.invalidItem`? could see this cropping up in different contexts?